src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -32,13 +38,21 @@
 import io.flutter.console.FlutterConsoleHelper;
 import io.flutter.dart.DartPlugin;
 import io.flutter.utils.FlutterModuleUtils;
+import org.jdom.Element;
+import org.jdom.JDOMException;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+import java.io.File;
+import java.io.IOException;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class FlutterSdk {
   public static final String FLUTTER_SDK_GLOBAL_LIB_NAME = "Flutter SDK";
+
+  private static final String SDK_LIBRARY_PATH = ".idea/libraries/Dart_SDK.xml";
```

Comment:
Thanks for the reference; I'll see what I can do with StartupActivity.

Neither ProjectOpenProcessor nor StartupActivity have any javadoc. Could you add some? And maybe have cross-references between them. More javadoc would save a lot of time for plugin authors.